OPAL/si/src/main/scala/org/opalj/fpcf/PropertyStore.scala

```diff
@@ -135,8 +135,7 @@ abstract class PropertyStore {
     //
 
     private[this] val externalInformation = new ConcurrentHashMap[AnyRef, AnyRef]()
-
-    var currentPhaseConf: Option[PhaseConfiguration[_]] = None
+    protected[this] var currentPhaseToDelete: Set[Int] = Set.empty
```

Comment:
Consider to make this consistent with other phase configurations, i.e., an array of Boolean values.